python/ray/internal/internal_api.py

```diff
@@ -39,6 +39,40 @@ def memory_summary(node_manager_address=None, node_manager_port=None):
     stub = node_manager_pb2_grpc.NodeManagerServiceStub(channel)
     reply = stub.FormatGlobalMemoryInfo(
         node_manager_pb2.FormatGlobalMemoryInfoRequest(), timeout=30.0)
+    return reply.memory_summary + "\n" + store_stats_summary(reply)
+
+
+def node_stats(node_manager_address=None, node_manager_port=None):
```

Comment:
Is it copied from dashboard code? Can we just share the same method for this?

Reply:
Actually, this isn't copied from the dashboard. It's implemented in a similar way as memory_summary(), but enables us to get those core stats in a dictionary rather than a large string.